conf/defaults.ini

```diff
@@ -1502,6 +1502,8 @@ public_key_retrieval_disabled = false
 public_key_retrieval_on_startup = false
 # Enter a comma-separated list of plugin identifiers to avoid loading (including core plugins). These plugins will be hidden in the catalog.
 disable_plugins =
+# Auth token for plugin installations and removal in managed instances
+install_token =
```

Comment:
can you verify if is it okay to put a token in configuration? Maybe we have a better place for this?

Reply:
That is how gcom provide tokens to the instances and that's how the feature toggles admin area doing:
https://github.com/grafana/hosted-grafana/blob/main/pkg/hgapi/instance/config_ini.go#L491-L495

https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L1707